### ISSUE TITLE
Update LXD docker profile to load 'overlayfs' kernel module in order to provide 'overlay' Docker storage driver

### ISSUE THREAD

**Issue by atc0005** [atc0005/ansible-role-lxd-testenv#9]
See also #4, #8.

Per https://github.com/atc0005/ansible-role-lxd-testenv/issues/8#issuecomment-461516791:

> `lxc config set <container> linux.kernel_modules overlay`

Within the container:

When using `docker-ce` version `18.09.1~3-0~ubuntu-xenial` both `overlay` and `overlay2` Docker storage drivers are available (at least when using `dir` LXD storage).

**Comment by atc0005** [comment]
> When using `docker-ce` version `18.09.1~3-0~ubuntu-xenial` both `overlay` and `overlay2` Docker storage drivers are available (at least when using `dir` LXD storage).

Not true. I thought I remembered that the `overlay2` Docker storage driver wasn't an option with LXD and it turns out it isn't specific to LXD 2.0:

`root@ansible-ubuntu-docker:~# docker pull ubuntu:16.04`

```ShellSession
16.04: Pulling from library/ubuntu
7b722c1070cd: Pull complete 
5fbf74db61f1: Pull complete 
ed41cb72e5c9: Extracting [==================================================>]     526B/526B
7ea47a67709e: Download complete 
failed to register layer: Error processing tar file(exit status 1): operation not permitted
```

**Comment by atc0005** [comment]
Needed changes have been applied to a branch I'm working on as part of a cleanup effort for v1.1; this will be included in the next PR.

**Comment by atc0005** [comment]
Included in #14.
